src/parsers/chase-credit-card-parser.ts

```diff
@@ -63,8 +63,13 @@ function performStateAction(currentState: State, line: string, yearPrefix: numbe
         const accountNumberMatch = line.match(/account number: .+(\d{4})$/i);
         if (closingDateMatch) {
             const [, startDateString, endDateString] = closingDateMatch;
-            output.startDate = dateFromSlashFormat(startDateString, yearPrefix);
-            output.endDate = dateFromSlashFormat(endDateString, yearPrefix);
+            const startDate = dateFromSlashFormat(startDateString, yearPrefix);
+            const endDate = dateFromSlashFormat(endDateString, yearPrefix);
+            // Chase statements sometimes include transactions a few days outside of the statement range.
```

Comment:
Adding this logic to `processTransactionLine` instead, so the statement date's remains unchanged.